script/Arcueid/Day 2/02_00_ARC02_1_1.txt

```diff
@@ -114,7 +114,7 @@ I'm always open to questions, so feel free to come after me, okay ♡"
 [sha:e338152ddbce642546bd1f46a2ecd875eb398c2b]{
 -- Page 133, Offset 2948.
 -- がたり、とあさましくも不穏な物音が教室に木霊した。
-A jumble of crashing and banging sounds echo through the classroom.
+A jumble of crashing and banging sounds echoes through the classroom.
```

Comment:
A single sound echoes, but multiple sounds echo. English doesn't make any sense!

Reply:
(To be clear, that means revert this)